packages/access-api/.dev.vars

```diff
@@ -0,0 +1,2 @@
+SENTRY_DSN="https://f8b04d1241bb4b618443b7cc269ee6c1@o609598.ingest.sentry.io/6536511"
```

Comment:
I tried to gitignore this :/ I disabled this DSN in the sentry admin once I saw this.